tests/finance/test_slippage.py

```diff
@@ -1146,3 +1147,85 @@ def test_orders_stop(self, name, order_data, event_data, expected):
 
                 for key, value in expected['transaction'].items():
                     self.assertEquals(value, txn[key])
+
+
+class FixedBasisPointsSlippageTestCase(WithCreateBarData,
+                                       WithSimParams,
+                                       ZiplineTestCase):
+
+    START_DATE = pd.Timestamp('2006-01-05 14:31', tz='utc')
+    END_DATE = pd.Timestamp('2006-01-05 14:36', tz='utc')
+    SIM_PARAMS_CAPITAL_BASE = 1.0e5
+    SIM_PARAMS_DATA_FREQUENCY = 'minute'
+    SIM_PARAMS_EMISSION_RATE = 'daily'
+
+    ASSET_FINDER_EQUITY_SIDS = (133,)
+    ASSET_FINDER_EQUITY_START_DATE = pd.Timestamp('2006-01-05', tz='utc')
+    ASSET_FINDER_EQUITY_END_DATE = pd.Timestamp('2006-01-07', tz='utc')
+    minutes = pd.DatetimeIndex(
+        start=START_DATE,
+        end=END_DATE - pd.Timedelta('1 minute'),
+        freq='1min'
+    )
+
+    @classproperty
+    def CREATE_BARDATA_DATA_FREQUENCY(cls):
```

Comment:
Since we're not actually using sim params in the test here, I would probably just set this attribute directly at class scope and remove the dependency on sim_params entirely.

Reply:
removed SimParams